webAO/client.js

```diff
@@ -90,6 +90,46 @@ if (window.requestIdleCallback) {
 
 let lastICMessageTime = new Date(0);
 
+const onMessage = (e) => {
```

Comment:
these functions belong in the Client class. use `bind()` function to preserve the `this` context